website/docs/tutorial/1b-setting-up-snowflake.md

```diff
@@ -3,14 +3,299 @@ title: Setting up with Snowflake
 id: setting-up-snowflake
 ---
 
-This guide is under construction
-
 ## Get started with Snowflake
 
+1. To get started, sign up for a free trial by following [this link](https://signup.snowflake.com/). 
+
+2. Then fill out the signup form.
+
+<p align="center">
+<Lightbox src="/img/snowflake_tutorial/snowflake_free_trial_signup_form.png" title="Snowflake Free Trial Signup Form" />
+</p>
+
+3. Select the Enterprise edition, choose a cloud provider and region, and agree to the terms of service. There are some organizational questions to think about when choosing a cloud provider for a full implementation, which you can read more about [here](https://docs.snowflake.com/en/user-guide/intro-cloud-platforms.html). For the purposes of this setup, all cloud providers and regions will work so choose whichever you’d like. Finally, click "GET STARTED".
+
+<p align="center">
+<Lightbox  src="/img/snowflake_tutorial/snowflake_account_configuration.png" title="Snowflake Account Configuration" />
+</p>
+
+4. After submitting the signup form you should receive an email asking you to activate your account. Click on the link in the email and a new tab will open up where you’ll create your username and password. Enter in all that info and click "Get started".
+
+<p align="center">
+<Lightbox src="/img/snowflake_tutorial/snowflake_account_signup.png" title="Snowflake Account Signup" />
+</p>
+
+5. Congrats! Your workspace is ready for some data. Feel free to check out any of the getting started tooltips that Snowflake provides in the UI to familiarize yourself before moving on to the next section.
+
+<p align="center">
+<Lightbox src="/img/snowflake_tutorial/snowflake_workspace.png" title="Snowflake Workspace" />
+</p>
+
+
 ## Load sample data into Snowflake
 
-## Set up a project to connect dbt Cloud to Snowflake (partner connect)
+1. Now we’re ready for some sample data. The data used here is stored as CSV files in a public S3 bucket and the following steps will guide you through how to prepare your Snowflake account for that data and upload it.
+
+2. Run the following commands to create a new virtual warehouse, two new databases (one for raw data, the other for future dbt development), and a `jaffle_shop` schema for your raw data. Feel free to copy/paste from below:
+
+```sql
+create warehouse transforming;
+
+create database raw;
+
+create database analytics;
+
+create schema raw.jaffle_shop;
+```
+
+<p align="center">
+<Lightbox src="/img/snowflake_tutorial/snowflake_create_warehouse_database_and_schema_commands.png" title="Snowflake - Create Warehouse, Database, and Schema Commands" />
```

Comment:
new screenshot for this one if you want to use it 
<img width="372" alt="image" src="https://user-images.githubusercontent.com/10554338/152814259-e0f58078-6d5b-4c08-a77d-0e0474db6272.png">

Reply:
@coapacetic We might want to change over to the new UI because it's the standard UI you land in with new accounts. I just recently have to update my quickstart fort his reason

Reply:
^^ that's what I was thinking - @bobbybirstockdbtlabs let us know if you need help grabbing some new screenshots / steps.  We likely want to tell the learner to make a new worksheet, etc.
<img width="1437" alt="image" src="https://user-images.githubusercontent.com/10554338/152832823-895ac84b-daaa-42c2-8e6a-8996b5be62ab.png">

Reply:
Yup, I can definitely switch this up and take new screenshots. It's interesting because for whatever reason, my trials still default to the classic view and make me login again to be able to see everything in "Snowsight". But I'd imagine this will change soon for everyone, better to have them updated now.